kolibri/core/views.py

```diff
@@ -84,7 +85,12 @@ def set_language(request):
 
 def logout_view(request):
     logout(request)
-    return HttpResponseRedirect(reverse("kolibri:core:redirect_user"))
+    if LogoutRedirectHook.registered_hooks:
```

Comment:
Mea culpa - I misled you here, `registered_hooks` is a generator, so this will always be True, need to do a `len` on it.

Reply:
solved (I think)